encfs/RawFileIO.cpp

```diff
@@ -220,14 +221,17 @@ bool RawFileIO::write(const IORequest &req) {
   ssize_t bytes = req.dataLen;
   off_t offset = req.offset;
 
+  int eno = 0;
   while ((bytes != 0) && retrys > 0) {
+    errno = 0;
```

Comment:
instead of setting errno, can we eno into the "writeSize < 0" branch?  pwrite will return -1 if it sets errno.